model/context.go

```diff
@@ -12,32 +12,34 @@ import (
 // function, all the other applicable fields in the Context can
 // inferred and populated from the id of any one of the fields.
 type Context struct {
-	Task       *task.Task
-	Build      *build.Build
-	Version    *Version
-	Patch      *patch.Patch
-	ProjectRef *ProjectRef
-
-	project *Project
+	Task    *task.Task
+	Build   *build.Build
+	Version *Version
+	Patch   *patch.Patch
+
+	projectRef     *ProjectRef
```

Comment:
I think this needs to be public because parts of it are used in HTML templates

Reply:
I don't see any places where we pass the context to the template directly.

Reply:
JK.